spec/h3_spec.rb

```diff
@@ -550,4 +550,36 @@
       end
     end
   end
+
+  describe ".origin_h3_indexes_from_unidirectional_edge" do
+    let(:h3_index) { "11928308280fffff".to_i(16) }
+    let(:expected_indexes) do
+      %w(8928308280fffff 8928308283bffff).map { |i| i.to_i(16) }
+    end
+
+    subject(:origin_h3_indexes_from_unidirectional_edge) do
+      H3.origin_h3_indexes_from_unidirectional_edge(h3_index)
+    end
+
+    it "has two expected h3 indexes" do
+      expect(origin_h3_indexes_from_unidirectional_edge).to eq(expected_indexes)
+    end
+  end
+
+  describe ".h3_unidirectional_edges_from_hexagon" do
+    let(:h3_index) { "11928308280fffff".to_i(16) }
```

Comment:
I'm surprised this works. That's a unidirectional edge index, not a regular H3 index. You can't get an edge traversal from an edge traversal, but from a hexagon to all of its immediate neighbors. Most likely the code is using the origin hexagon here based on how unidirectional edge indexes work?

Reply:
Not sure - I'll change this spec to ensure it uses a regular h3 index. Good catch!